### PR TITLE
Use ** operator over function pow()

### PR DESCRIPTION
**Pull request by SpacePossum**
With Twig PHP dep. raised to 5.6 the arithmetic operator [`**`](http://php.net/manual/en/language.operators.arithmetic.php) can be used.

Besides slightly faster it is also harder to override an operator than it is the `pow()` function.

### CODE REVIEW

File: lib/Twig/Node/Expression/Binary/Power.php [hunk 1]
```diff
@@ -13,11 +13,9 @@ class Twig_Node_Expression_Binary_Power extends Twig_Node_Expression_Binary
     public function compile(Twig_Compiler $compiler)
     {
         $compiler
-            ->raw('pow(')
             ->subcompile($this->getNode('left'))
-            ->raw(', ')
+            ->raw(' ** ')
             ->subcompile($this->getNode('right'))
```
Comment (stof) [inline lib/Twig/Node/Expression/Binary/Power.php]: You can actually remove the ``compile`` method entirely, as the parent implementation will already do things properly.

Comment (SpacePossum) [inline lib/Twig/Node/Expression/Binary/Power.php]: awesome :)  :+1:

**Comment by stof** [comment]
The tests should be added in 1.x (btw, are they really missing ?)

**Comment by SpacePossum** [comment]
test for this is in https://github.com/twigphp/Twig/pull/2255

**Comment by fabpot** [comment]
master is Twig 3.x, so this should be switched to 2.x.

**Comment by fabpot** [comment]
Thank you @SpacePossum.

**Pull request closed without merging**